credmaster.py

```diff
@@ -4,707 +4,965 @@
 from utils.fire import FireProx
 import utils.utils as utils
 import utils.notify as notify
+import utils.mfasweep as mfasweeper
```

Comment:
There are only three places where code needs to be updated (input args, arg parsing, run mfasweep), I appreciate you taking the time to do formatting but it isn't necessary

Reply:
just wanted it to be tidy :)